### PR TITLE
Feat: add red underline annotator for maven vulnerabilities

### PR DESCRIPTION
**Pull request by aldanchenko**
This will add a red underline for Maven vulnerabilities.

Screen Shot 2020-01-27 at 6 51 09 PM

### CODE REVIEW

File: src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      val vulnerabilityName = securityVulnerability.name
+      val vulnerabilityNameParts = vulnerabilityName.split(":")
+
+      vulnerabilityNameParts.foreach(name => {
+        if (element.getText == name) {
+          import com.intellij.openapi.util.TextRange
+
+          val range = new TextRange(element.getTextRange.getStartOffset, element.getTextRange.getEndOffset)
+          holder.createErrorAnnotation(range, "Vulnerability issue")
```
Comment (yuliabaron) [inline src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala]: Where do I see this error text?

Comment (aldanchenko) [reply src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala]: When a user hovers over dependency text.

Screen Shot 2020-01-28 at 4 19 43 PM

I think we can add a more useful description. For example, in this case, show "XML External Entity (XXE) Injection" message.

Comment (yuliabaron) [reply src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala]: Maybe then `Vulnerable package`?

Comment (aldanchenko) [reply src/main/scala/io/snyk/plugin/ui/annotators/SnykMavenRedUnderlineAnnotator.scala]: Ok. I'll change it.

**Pull request merged**